src/components/buttons/button/styles.ts

```diff
@@ -1,26 +1,15 @@
 import styled, { css, CssWithTheme } from 'styled-components'
-import { StyledButtonProps } from './index'
 import { ButtonSize } from './enums'
 
-const { Normal } = ButtonSize
-
-function getDefaultColorStyle(disabled: boolean) {
-  if (disabled) {
-    return css`
-      background: ${({ theme }) => theme.colors.gravel};
-      color: ${({ theme }) => theme.colors.white};
-    `
-  }
-
-  return css`
-    &:hover {
-      background-color: ${({ theme }) => theme.colors.darkIt};
-    }
-    background-color: ${({ theme }) => theme.colors.it};
-    color: ${({ theme }) => theme.colors.white};
-  `
+export interface StyledButtonProps {
+  // https://styled-components.com/docs/api#transient-props
+  $inverted?: boolean
```

Comment:
![Screen Shot 2021-02-07 at 1 06 18 PM](https://user-images.githubusercontent.com/15198653/107147686-a49cbb80-694f-11eb-8c56-76ade187b039.png)

Reply:
This is OK to me. 👍